### PR TITLE
Update Arrays BBE for slp9

### PR DESCRIPTION
**Pull request by KRVPerera**
## Purpose
Array reverse is now inline. Also how to use a key when sorting is not clear in the example.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/arrays/arrays.bal [hunk 1]
```diff
@@ -14,6 +14,7 @@ public function main() {
     // Arrays support several inbuilt functions such as `.reverse()`, `.pop()`, `.push()`, and `.removeAll()`.
     int[] bReveresed = b.reverse();
     io:println("Reversed: ", bReveresed);
+    io:println("Reversed array and original array is exact : ", bReveresed === b);
```
Comment (MaryamZi) [inline examples/arrays/arrays.bal:17]:
```suggestion
    io:println("Reversed array and original array are the same: ", bReveresed === b);
```

File: examples/arrays/arrays.bal [hunk 2]
```diff
@@ -22,25 +23,27 @@ public function main() {
...
-                return ["A",value.toString()];
-            } else {
-                return ["B",value.toString()];
-            }
-        });
-    io:println("Sorted Array: ", sortedArray);
+    int[] sortedArray = b.sort();
+    io:println("Array sorted in ascending order: ", sortedArray);
+
+    sortedArray = b.sort(array:DESCENDING, isolated function(int value) returns string[] {
```
Comment (MaryamZi) [inline examples/arrays/arrays.bal:39]: Isn't the character count > 80 here? 

https://github.com/ballerina-platform/ballerina-distribution/blob/master/doc-guidelines.md#ballerina-by-examples-guidelines

File: examples/arrays/arrays.bal [hunk 3]
```diff
@@ -22,25 +23,27 @@ public function main() {
...
+                                                     if (value < 5) {
+                                                         return ["Z", value.toString()];
+                                                     } else {
+                                                         return ["B", value.toString()];
+                                                     }
+                                                 });
+    io:println("Array sorted in descending order using a key : ", sortedArray);
```
Comment (MaryamZi) [inline examples/arrays/arrays.bal:46]:
```suggestion
    io:println("Array sorted in descending order using a key: ", sortedArray);
```

**Review by MaryamZi — CHANGES_REQUESTED** [review]

File: examples/arrays/arrays.bal (hunk 2 above)
Comment (KRVPerera) [reply examples/arrays/arrays.bal]: Fixed it by reverting to previous formatting.
3391dd21aeb9e78228947d40e8436a90f226304a

**Review by MaryamZi — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**